connect/src/state.rs

```diff
@@ -233,6 +233,22 @@ impl ConnectState {
         self.request.is_active
     }
 
+    /// Returns the `is_playing` value as perceived by other connect devices
+    ///
+    /// see [ConnectState::set_status]
+    pub fn is_playing(&self) -> bool {
+        let player = self.player();
+        player.is_playing && !player.is_paused
+    }
+
+    /// Returns the `is_paused` state value as perceived by other connect devices
+    ///
+    /// see [ConnectState::set_status]
+    pub fn is_pause(&self) -> bool {
+        let player = self.player();
+        player.is_playing && player.is_paused && player.is_buffering
```

Comment:
As a sanity check: so if we are 100% buffered, we cannot be in pause state?

Reply:
Hmm, my method naming might be a bit misleading as the player here isn't the actual player and rather the state of the player that the server knows of.

And by that these bool flags are all independent from the actual player logic. Which also means we can set them as we like to, and for pausing the mobile and desktop want all of these flags set to true. I can't tell you why, but setting them to the expected values (so only pause would be true if we are pausing) we can't resume playback as the play button will be disabled.

I have written down that explanation somewhere by the reference method.

But yeah as a mixture of my confusing naming and Spotify's confusing expectations this looks rather wrong^^;